docs/source/getting-started/database.rst

```diff
@@ -1,7 +1,7 @@
 Database Setup
 ===============
 
-One of the reasons that Augur is so powerful is because of our `unified data model <../schema/data-model.html>`_.
+One of the reasons that Augur is so powerful is because of our `unified data model <https://oss-augur.readthedocs.io/en/dev/architecture/data-model.html>`_.
```

Comment:
This link to the schema model will actually not work once this gets merged into `dev`. Our documentation is built by branch, and this link points to the version that's currently in `dev`. Because of this, the page the link points to will get deleted when the new documentation is merged into `dev`, and since that page would not exist the link would be defunct again. We should either get rid of the link entirely and just have it be plain text, or re-implement the page in its new form under `docs/source/schema/`.

Reply:
I think implementing it in schema is a better option because the link provides really useful information? Can I go ahead and implement it in schema?

Reply:
I see you already started, which is great!